diceRoll/diceRoll2.js

```diff
@@ -333,31 +326,23 @@ function App(context) {
     },
 
     // functions for displaying data
-    toggleSaved: function(override) {
```

Comment:
This function was heavily overburdened and refactored to much more digestible chunks